.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Add 
`# RubyMine`
`.idea`
to your .gitignore file to ignore this RubyMine generated folder/files.